src/backend/marsha/core/tests/utils/test_medialive_utils.py

```diff
@@ -1287,3 +1290,53 @@ def test_delete_mediapackage_channel(self):
             deleted_endpoints = medialive_utils.delete_mediapackage_channel("1")
             mediapackage_client_stubber.assert_no_pending_responses()
         self.assertEqual(deleted_endpoints, ["1", "2"])
+
+    def test_update_id3(self):
+        """Should update id3 tags on a video channel."""
+        video = VideoFactory(
+            live_info={
+                "medialive": {
+                    "input": {"id": "medialive_input1"},
+                    "channel": {"id": "medialive_channel1"},
+                },
+                "mediapackage": {
+                    "endpoints": {
+                        "hls": {"id": "mediapackage_endpoint1"},
+                    },
+                    "channel": {"id": "mediapackage_channel1"},
+                },
+            }
+        )
+        serialized_id3_video = VideoId3TagsSerializer(video)
+
+        now = datetime(2018, 8, 8, tzinfo=timezone.utc)
+        with mock.patch("datetime.datetime") as mock_dt, Stubber(
+            medialive_utils.medialive_client
+        ) as mediapackage_client_stubber:
+            mock_dt.now = mock.Mock(return_value=now)
+            mediapackage_client_stubber.add_response(
+                "batch_update_schedule",
+                service_response={},
+                expected_params={
+                    "ChannelId": "medialive_channel1",
+                    "Creates": {
+                        "ScheduleActions": [
+                            {
+                                "ActionName": now.isoformat(),
+                                "ScheduleActionStartSettings": {
+                                    "ImmediateModeScheduleActionStartSettings": {}
+                                },
+                                "ScheduleActionSettings": {
+                                    "HlsId3SegmentTaggingSettings": {
+                                        "Tag": json.dumps(
+                                            {"video": serialized_id3_video.data}
```

Comment:
You should not use computed data but exactly what you expect. Otherwise you will not detect future regression.